.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+{
+  "build": {
+    "context": "${localWorkspaceFolder}/.devcontainer",
+    "dockerfile": "${localWorkspaceFolder}/.devcontainer/Dockerfile",
+    "args": {
+      "CUDA": "11.8",
+      "PYTHON_PACKAGE_MANAGER": "conda",
+      "BASE": "rapidsai/devcontainers:25.04-cpp-cuda11.8-mambaforge-ubuntu22.04"
```

Comment:
We'll need to copy in all the `ci/release/update-version.sh` logic that we use to support this. It'll need to be different for ucxx than most libraries due to the version scheme differences. Note that both full (`25.04`) and PEP 440 (`25.4`) versions are needed in this file.

Reply:
🤦🏻 you're right of course, sorry for not being more careful about this.

Just pushed https://github.com/rapidsai/ucxx/pull/363/commits/7980f3c0600c2d1544e3f5ccdf683f840e80b024 with fixes.

Tested like this:

```shell
./ci/release/update-version.sh '0.44.00'
git grep -E '25\.2|25\.02|25\.4|25\.04|0\.42|0\.43'
```